rocketpy/simulation/monte_carlo.py

```diff
@@ -518,7 +522,9 @@ def __evaluate_flight_outputs(self, flight, sim_idx):
                     ) from e
             outputs_dict = outputs_dict | additional_exports
 
-        return json.dumps(outputs_dict, cls=RocketPyEncoder) + "\n"
+        return (
+            json.dumps(outputs_dict, cls=RocketPyEncoder, **self._export_config) + "\n"
```

Comment:
This change had already been done by PR #679, however is was likely undone by merging errors at some point.